fridge/src/main/java/ua/lviv/iot/algo/part1/lab1/services/WineFridgeService.java

```diff
@@ -0,0 +1,48 @@
+package ua.lviv.iot.algo.part1.lab1.services;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+import ua.lviv.iot.algo.part1.lab1.models.WineFridge;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Service
+@Scope("singleton")
+public class WineFridgeService {
+    private final Map<Integer, WineFridge> map = new HashMap<>();
+    private Integer nextAvailable = 1;
+
+    public Collection<WineFridge> getAllFridges() {
+        return this.map.values();
+    }
+
+    public WineFridge getFridgeById(Integer id) {
+        return this.map.get(id);
+    }
+
+    public WineFridge addFridge(WineFridge entity) {
+        entity.setId(this.nextAvailable);
+        this.map.put(this.nextAvailable, entity);
+        this.nextAvailable++;
+        return this.map.get(this.nextAvailable - 1);
```

Comment:
`      entity.setId(this.nextAvailableId++);
        this.map.put(entity.getId(), entity);
        return entity;`